dynamics/apis/api_base.py

```diff
@@ -143,6 +144,8 @@ def _get_request_for_count(self, params, api_url):
         if response.status_code == 200 or response.status_code == 201:
             raw_content = response.content
             result = raw_content.decode('utf-8')
+            result = result.lstrip('\ufeff')
```

Comment:
have confirmed this

if you decode i don't think this is needed
Can you verify once